frontend/hooks/useFileStorage.js

```diff
@@ -67,9 +151,16 @@ export default function useFileStorage(
     isFetching,
     isPending,
     isPlaceholderData,
-    error,
-    deleteItem,
-    deleteError: deleteMutation.error,
-    deleteSuccess: deleteMutation.isSuccess,
+    isError,
+    defaultError: error,
+    deleteItem: (item) => deleteMutation.mutateAsync(item),
```

Comment:
(asking) should this be:
`    deleteItem: (item) => deleteMutation.mutateAsync({item}),`